src/App.jsx

```diff
@@ -1,7 +1,10 @@
 import "./App.css";
 
+import { ThemeProvider, createGlobalStyle } from "styled-components";
+import { theme } from "./styles/styled/theme/theme";
+
 function App({ children }) {
-  return <>{children}</>;
+  return <ThemeProvider theme={theme}>{children}</ThemeProvider>;
```

Comment:
```suggestion
  return <ThemeProvider theme={theme}>{children}</ThemeProvider>;
```
ThemeProvider역할을 생각해보면 전역으로 theme prop을 공급해주는데, 

App에서 공급하는것보단,  index.js에서 ThemeProvider가 Router를 감싸는 식으로 공급해주는게 
한번만 설정하고 전역에 공급할 수 있는 적절한 방법인것같습니다.

예를들면 이렇게요 :-) 

```jsx
<ThemeProvider>
    <Router/>
</ThemeProvider>
```